### DISCUSSION TITLE
How to configure appStartPath

### DISCUSSION THREAD

**Discussion by aldo-roman** [ionic-team/capacitor#6976] [Q&A]
Hello!

I need my app to load in `capacitor://localhost/` (with a slash at the end).

I see there is an `appStartPath` variable in the files, but I cannot find where in the configuration should I set this to `/`.

https://github.com/ionic-team/capacitor/blob/79e17bb5e6ccd813bddc626703152d3983f6d93b/ios/Capacitor/Capacitor/CAPInstanceConfiguration.swift#L11-L16

How can I achieve this?

**Comment by jcesarmobile** [comment]
why do you need the slash at the end? it should behave the same way with or without it

**Reply by aldo-roman** [reply]
Yes the app opens, but I need to load `/` first, so that my app can detect the "activeUrl" and the relevant UI is displayed as "home". The property is there in the code, but it seems the descriptor is not reading from the config.

I also get my analytics traced as `capacitor://localhost` (first launch) and `capacitor://localhost/` (subsequent clicks on the home icon) which I find inconvenient.

I am using SvelteKit, FWIW

I can imagine other use cases where a dev would like the app to open somewhere else than an empty path.

PS: Seems like this is a bug/feature request, not a discussion :pray:

**Reply by ChangYuJonathanWu** [reply]
I'm in this exact situation. I want to open the app at `/login`. While I could conditionally render the right components at `/` by detecting if a native platform is used, I'm using SSR on my home page and want to preserve the SEO and performance benefits. 

Alternatives like redirecting to the right page based on client-side logic (e.g. detecting if native platform) are slow and will still show the original page before the redirect is complete.

**Reply by cadlagtrader** [reply]
Same issue for me, using RedwoodJS
